pkgs/development/libraries/science/math/openblas/default.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, gfortran, perl, liblapack, config }:
+{ stdenv, fetchurl, gfortran, perl, liblapack, config, coreutils, clang }:
```

Comment:
hmm, rather than omitting it I should probably put it in as a build input. I suppose it's able to find it regardless, but otherwise it seems there's a risk of it failing (esp. if the clang version is too old).